drivers/display/ili9340/Kconfig

```diff
@@ -0,0 +1,88 @@
+# Kconfig - ILI9340 display driver configuration options
+
+#
+# Copyright (c) 2017 dXplore
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ILI9340
+	bool "ILI9340 display driver"
+	depends on DISPLAY && SPI
+	default n
+	help
+	Enable driver for ILI9340 display driver.
+
+config ILI9340_DEV_NAME
+	string "ILI9340 device name"
+	depends on ILI9340
```

Comment:
you could have an if ILI9340 ... endif # ILI9340 all around these config options, it would avoid adding depends on ILI9340.